src/TraceEvent/Ctf/CtfTracing.Tests/CtfTracing.Tests.csproj

```diff
@@ -36,19 +36,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework" />
   </ItemGroup>
-  <Choose>
```

Comment:
I am assuming that the tests still run and pass.   Also I need this to work in 2015 too.